### DISCUSSION TITLE
Help passing the CI tests.

### DISCUSSION THREAD

**Discussion by theanmolsharma** [checkstyle/checkstyle#9624] [Q&A] [answered]
Hello fellow contributors. I am Anmol Sharma, a developer from India. I recently started contributing to open source. Therefore I need some guidance. I am trying to fix #9549 . I have made the necessary changes and my changes are approved but I am unable to pass the CI tests. Could anyone please help me?

**Comment by romani** [comment, marked as answer]
resolved - https://github.com/checkstyle/checkstyle/pull/9612
